blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,228 @@
+:root {
+  --border-color: var(--c-grey-300);
+}
+
+.v2-footer {
+  background: var(--c-background-card);
+}
+
+.v2-footer :is(h3, p, a) {
+  font-family: var(--ff-volvo-novum);
+  font-feature-settings: 'clig' off, 'liga' off;
+  font-size: 16px;
+  letter-spacing: 0.5px;
+}
+
+.v2-footer ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.v2-footer-links-wrapper :is(h3, li, p, a) {
+  margin-bottom: 0;
+  color: var(--c-main-black);
+}
+
+.v2-footer-list h3 {
+  font-weight: bold;
+  border-top: 1px solid var(--border-color);
+  cursor: pointer;
+  padding: 28px 16px;
+  display: flex;
+  align-items: center;
+}
+
+.v2-footer-list:not(.expand) h3:hover {
+  background-color: #B8DeD8;
+}
+
+.v2-footer-list-item {
+  transition: 300ms max-height ease;
+  will-change: max-height;
+  overflow: hidden;
+  max-height: 0;
+}
+
+.v2-footer-list-item a {
+  display: flex;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 16px;
+}
+
+.icon-accordion {
+  margin-left: auto;
+}
+
+.icon-newtab,
+.icon-fb,
+.icon-twitter,
+.icon-yt,
+.icon-instagram,
+.icon-linkedin {
+  display: none;
+}
+
+.v2-footer-seperator {
+  height: 1px;
+  background-color: var(--border-color);
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.v2-footer-copyright-wrapper {
+  display: flex;
+  padding: 12px 0;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 6px;
+}
+
+.v2-footer-bar {
+  display: flex;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.v2-footer-bar a,
+.v2-footer-copyright p {
+  color: var(--volvo-text-gray);
+  margin: 0;
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+.v2-footer a,
+.v2-footer a:hover {
+  text-decoration: none;
```

Comment:
In [the prototype](https://www.figma.com/proto/IcqreRKnNYyDVMc28yoOeh/Homepage?page-id=2%3A17&node-id=1461-18527&scaling=scale-down&starting-point-node-id=875%3A12886&hide-ui=1) I do see text-decoration on hover

Reply:
I don't see text-decoration on hover 
<img width="1459" alt="Screenshot 2023-08-08 at 13 32 52" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/4208604/c59805a4-9e44-4dd4-8216-4cdccd9fc1e2">

Reply:
I don't see the text-decoration
<img width="1459" alt="Screenshot 2023-08-08 at 13 32 52" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/4208604/275467f7-5c73-4e95-82e4-ad9cbd1f230e">